src/olympia/addons/tests/test_admin.py

```diff
@@ -409,15 +448,15 @@ def test_query_count(self):
         self.grant_permission(user, 'Addons:Edit')
         self.grant_permission(user, 'Admin:Advanced')
         self.client.login(email=user.email)
-        with self.assertNumQueries(23):
+        with self.assertNumQueries(24):
             # It's very high because most of AddonAdmin is unoptimized but we
             # don't want it unexpectedly increasing.
             response = self.client.get(self.detail_url, follow=True)
         assert response.status_code == 200
         assert addon.guid in response.content.decode('utf-8')
 
         version_factory(addon=addon)
-        with self.assertNumQueries(23):
+        with self.assertNumQueries(24):
```

Comment:
this would be 1 higher if you added a version of a different channel but I guess that's still okay